themes/ddbasic/sass/configuration/_mixin.scss

```diff
@@ -0,0 +1,189 @@
+//
+// Create mixins
+
+// Configuration
+//
+// Styleguide 3
+
+// Mixins
+//
+// Styleguide 3.1
+
+// ==========================================================================
+// Add-ons for Bourbon Neat
+// ==========================================================================
+
+// Adds an actual width the the Bourbon Neat outer-container mixin.
+//
+@mixin wrapper() {
+  @include outer-container;
+  width: $wrapper-width;
+  //width: $outer_width;
+}
+
+// Omega reset
+@mixin omega-reset($nth, $columns) {
+  &:nth-child(#{$nth}) { margin-right: flex-gutter($columns); }
+  &:nth-child(#{$nth}+1) { clear: none; }
+}
+
+// ==========================================================================
+// Mixins for placing icon and button with icon
+// ==========================================================================
+
+// Place-icon
+//
+// Mixin for placing icon on element
+//
+// place-icon($icon, $color, $size)
+//
+// $icon  - Name of icon to be placed - is a string, see _mixin.scss for icon names
+// $color - Color of icon - default is $charcoal-opacity-dark
+// $size  - Size of icon - default is 54px
+//
+// Styleguide 3.1.2
+
+@mixin place-icon(
+  $icon,
+  $color: $charcoal-opacity-dark,
+  $size: 54px) {
+  position: relative;
+  &::after {
+    @extend %icon-#{$icon};
```

Comment:
I think a mixin would be a better choice, but i think that's more of an opinon. Ok 👍